### ISSUE TITLE
Add possible ban warning in the README

### ISSUE THREAD

**Issue by TheSystemGuy1337** [irsat000/CAI-Tools#43]
This website is run by trigger-happy twats who love to abuse shadowbanning for no reason. (I was outright BANNED from their subreddit from suggesting alternatives! Thats how stupid and trigger-happy they are)

Because of this, it's safe to assume there is a non-zero chance they will outright IP ban you and terminate your account if you use this utility. A warning should be at the start which states this clearly. We cannot take chances. Even then, their "service" is on its way out and is dying rather rapidly. I can do this if you want,

**Comment by irsat000** [comment]
It's against EU regulations to use a number of ways to detect the running extensions. Even Youtube was not exempt to these rules when they tried to stop ad blocking extensions. And they are likely to not ban you but refuse service until you deactivated the extension, banning methods are bad for business. And there is also the fact that my extension is kind of insignificant to these giants.

Showing and talking about my extension can get your postings removed in their subreddit however, not outright ban normally. It usually falls under third party advertising.

Thank you for sharing your thoughts on this.

**Comment by TheSystemGuy1337** [comment]
> It's against EU regulations to use a number of ways to detect the running extensions. Even Youtube was not exempt to these rules when they tried to stop ad blocking extensions. And they are likely to not ban you but refuse service until you deactivated the extension, banning methods are bad for business. And there is also the fact that my extension is kind of insignificant to these giants.
> 
> Showing and talking about my extension can get your postings removed in their subreddit however, not outright ban normally. It usually falls under third party advertising.
> 
> Thank you for sharing your thoughts on this.

I doubt the people who run C.AI care, judging by the fact they permabanned me from their subreddit for suggesting alternatives, with the main one overall being better than C.AI in every category (that alternative was SillyTavern). They didn't even supply a reason for the permaban, so they banned me for no discernible reason. They also abuse shadow bans on their website and are very likely abusing shadow bans to silence criticism. If they actively abuse their power, it is certain there is a non-zero chance of them terminating accounts because someone used this extension. C.AI has just begun it's enshittification and it's only going to get worse. If they can't do this shit in the EU, then they will only target US users with bans for using this extension. Thats if they decide to pull a Discord and ban all third-party modifications (which I bet they will, judging by the fact their platform is a sinking ship). C.AI only cares about their investors and their bottom line, not the users. **You are not the customer; you are the product.**

In short, there is a non-zero chance they will crack down on people using this extension (at least in the United States). The risk grows with each passing day.

US consumer protection laws are almost nonexistent; companies can simply decide to screw you over whenever they feel like it and you will have no recourse. This is why I am considering moving to the UK or Canada. Just look at Nintendo; they screwed over legal preservation efforts, and made it illegal to mod your Switch in Japan for any legal reason (Ie running Linux or homebrew software)

**Comment by irsat000** [comment]
EU regulations must be implemented everywhere otherwise you can't service to EU at all. An example to this is the cookie policy, like clicking "Accept All Cookies", "Reject Cookies", yes that's EU regulation right there.

The reason you are perma banned in the subreddit is because it's third party advertising. It's rule 4 of their subreddit. That includes self-promoting which I did a year ago and perma banned also.

**Comment by TheSystemGuy1337** [comment]
> EU regulations must be implemented everywhere otherwise you can't service to EU at all. An example to this is the cookie policy, like clicking "Accept All Cookies", "Reject Cookies", yes that's EU regulation right there.
> 
> The reason you are perma banned in the subreddit is because it's third party advertising. It's rule 4 of their subreddit. That includes self-promoting which I did a year ago and perma banned also.
> 
> And since "they don't care about users", they also don't exert effort to ban people and detect the extension. It has its pros.

Even if that's the case; their platform is rapidly getting worse, and they will very likely go defunct and shut down in like 5 or 10 years when they can't turn a profit from it anymore or when people stop using the service and move on; this means in the end nobody wins. Fact of the matter, the investors will pull funding when the site is no longer profitable. Therefore, the service must get worse to remain profitable. Thats why YouTube tried banning adblock in the US, and that's why Google is introducing MV3 to kill off adblockers because it harms their bottom line. If C.AI was open source and community run and supported, then this conversation would have never happened. Thats not the case. I will however give you credit; your extension is great. Seeing that this platform is a sinking ship and will shut down soon, I used this extension as an archival tool. I only returned recently after leaving the platform for about a year (I abandoned it in 2022 to my memory) and coming back to it now it has gotten a lot worse. Also, there's this dice button next to the CAI Tools button that doesn't seem to do anything when clicked. What is it's purpose?

**Comment by irsat000** [comment]
It's for dragging the CAI Tools button. If you three tap on it, the button disappears. It's to make sure you always has access to it, immune to website changes. And to make sure it doesn't block anything beneath it, like the three dots or voice button.

**Comment by TheSystemGuy1337** [comment]
I didn’t know that, I thought it was static or a randomizer button as it looked like a dice. At least it doesn’t crash your browser, rickroll you or outright cause your PC to explode followed by the character you were chatting with to come into existence. UH OH, TERMINATOR PLOT!

And before we end up tanking the global economy, one tool at a time; we should release a freaking CD add on for it. That doesn’t work. *cough* Atari Jaguar CD *cough*

Either that or slap our name onto a cheap MP3 player as everyone was doing it in the 2000s, let’s get in on the game. Oh wait, that was 20 years ago.

**Comment by irsat000** [comment]
You have a screw loose, I love it.

**Comment by TheSystemGuy1337** [comment]
Mike, the whole things on fire. Next thing you know it I be calling up Microsoft tech support and saying “Hi, is this Microsoft tech support? I just slapped my name onto a cheap knockoff iPod, connected it to my PC and now everything is on fire. Can you help me?”

R.I.P TheSystemGuy, tried to call MS tech support after setting his own house on fire accidentally by slapping his own name onto a knockoff iPod, only for DankPods to smash it with the One Grit
????-Wish.com

Rest in Commodore Amiga, forever miss.

**Comment by TheSystemGuy1337** [comment]
CAI Tools CD add on is dead.

**Long live the CAI Tools CD add on.**
